aggregations/periodic_aggregations.py

```diff
@@ -2,7 +2,7 @@
 import datetime
 
 from .sql_aggregations import SqlAggregations
-from .db_tables import time_range_json
+from .db_tables import time_range_json, to_nanos
```

Comment:
I just want to point out the awkwardness of the methods `db_tables.time_range_json` and `db_tables.to_nanos` -- they are quite non-intuitive to me.